README.rst

```diff
@@ -1,12 +1,45 @@
-Baseband data input and output
-==============================
+Baseband: a Package for Radio Baseband I/O
+==========================================
 
 .. image:: http://img.shields.io/badge/powered%20by-AstroPy-orange.svg?style=flat
     :target: http://www.astropy.org
     :alt: Powered by Astropy Badge
 
-This is a package for reading and writing VLBI and other radio baseband
-files. It relies on numpy and astropy.  
+Baseband is a package for reading and writing VLBI and other radio baseband
+files, with the aim of simplifying and streamlining data conversion and
+standardization.  It relies on `NumPy <http://www.numpy.org/>`_ and `Astropy
+<http://www.astropy.org/>`_.
 
 For installation and usage instructions, please see the `online documentation 
 <https://baseband.readthedocs.io/>`_.
+
+Project Status
+--------------
+
+.. image:: https://travis-ci.org/mhvk/baseband.svg?branch=master
+
+.. image:: https://coveralls.io/repos/github/mhvk/baseband/badge.svg?branch=master
```

Comment:
The coverage one doesn't show anything if I just "view" this page - while travis does.

Reply:
I revised the badge, but apparently because it points to master, it won't properly display until this PR's been merged.